packages/client/src/modules/blocks/data-blocks/form-editor/fieldsInterfaceEditableSettings.tsx

```
@@ -0,0 +1,919 @@
+    //   ...fieldSchema?.['x-component-props']?.['fieldNames'],
+    // };
+    return {
+      async run() {
+        await form.submit();
+        const values = cloneDeep(form.values);
+        if (values.autoCreateReverseField) {
+          /* empty */
+        } else {
+          delete values.reverseField;
+        }
+        delete values.autoCreateReverseField;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)